README.rst

```diff
@@ -284,6 +341,15 @@ For example:
 
     CMIS Root > DRC > Melding Openbare Ruimte > 2020 > 12 > 31 > ZAAK-0000001 > document.txt
 
+**Relating a document to a Besluit**
+
+When a document is related to a Besluit, and that Besluit is part of a Zaak, the
+document is moved as if it was related to that Zaak.
```

Comment:
Not sure if we want to go in more details for the creations of object informatieobjects. These are the cases that I have described in the `drc_cmis/client.py` `create_oio()` function:

        1. The oio creates a link to a Besluit:
            A. The besluit is linked to a zaak:
                - If the document is NOT already linked to a zaak, it is moved from the temporary folder to
                    the zaak folder
                - If the document is already linked to a zaak, it will be copied to the new zaak folder
            B. The besluit is not linked to a zaak:
                - If the document is NOT already linked to a zaak, the oio is created in the temporary folder and the
                    document is NOT moved
                - If the document is already linked to a zaak, the document is not moved and the oio will go
                    in the temporary folder
        2. The oio creates a link to a Zaak:
            A. If the document is already related to a zaak, a copy of the document is put in the
                correct zaaktype/zaak folder.
            B. If the document is NOT related to a zaak: the document is moved from the temporary folder
            to the correct zaaktype/zaak folder.

        If the document is linked already to a gebruiksrechten, then the gebruiksrechten object is also moved.